packages/@storefront/flux-capacitor/src/flux-capacitor.ts

```diff
@@ -20,6 +20,24 @@ declare module 'redux' {
   }
 }
 
+export interface ParsedUrl {
+  route: string;
+  request: any;
+}
+export type Parse = (url: string) => Promise<void | ParsedUrl>;
+export type Build = (type: string, state: Store.State) => string;
+
+export const STOREFRONT_APP_ID = 'GroupBy StoreFront';
+
+export interface History {
+  parse: Parse;
+  build: Build;
+  pushState: (data: any, title: string, url: string) => void;
+  replaceState: (data: any, title: string, url: string) => void;
+  initialUrl: string;
+  noop?: () => void;
```

Comment:
Why do we define a noop here?

Reply:
Theoretically if someone wants to call these methods with a noop function they can use the one that's here. Idk I can take it out if you think it's not needed.